.golangci.yml

```diff
@@ -268,6 +268,10 @@ issues:
     - staticcheck
     text: "SA1019: in.(.+) is deprecated"
     path: .*(api|types)\/.*\/conversion.*\.go$
+  - linters:
+    - staticcheck
+    text: "SA1019: is deprecated in sets.String"
+    path: .*(cmd|controllers|controlplane|internal)\/.*\/*\.go$
```

Comment:
@ykakarap I am having issue finding why this regex is not catching the path

Reply:
path is okay (although I would drop the `\/*` in the middle. The problem is the text. The following should work:
```
  - linters:
    - staticcheck
    text: "SA1019: sets.String is deprecated"
    path: .*(cmd|controllers|controlplane|internal)\/.*\.go$
```

Reply:
Looks like an additional `make generate-modules` is required as well

Reply:
Then the final finding should be 
```
test/framework/namespace_helpers.go:147:31: Error return value of `eventInformer.AddEventHandler` is not checked (errcheck)
        eventInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Let's write the returned error into `err` + verify it's nil via: 
```go
Expect(err).NotTo(HaveOccurred())
```

Reply:
I'll take care of the follow-up.